src/AvtaleSide/NesteForrige/NesteForrige.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import { Route } from 'react-router';
+import { Context, medContext } from '../../AvtaleContext';
+import { pathTilStegIAvtale } from '../../paths';
+import { StegInfo } from '../AvtaleSide';
+import './NesteForrige.less';
+
+export interface Props {
+    avtaleSteg: StegInfo[];
+    aktivtSteg: StegInfo;
+}
+
+const finnForrigeSteg = (alleSteg: StegInfo[], steg: StegInfo) => {
+    const index = alleSteg.indexOf(steg);
+    if (index === 0) {
+        return null;
+    } else {
+        return alleSteg[index - 1];
+    }
+};
+
+const finnNesteSteg = (alleSteg: StegInfo[], steg: StegInfo) => {
+    const index = alleSteg.indexOf(steg);
+    if (index === alleSteg.length - 1) {
+        return null;
+    } else {
+        return alleSteg[index + 1];
+    }
+};
+
+const NesteForrige: FunctionComponent<Props & Context> = props => {
+    const forrigeSteg = finnForrigeSteg(props.avtaleSteg, props.aktivtSteg);
+    const nesteSteg = finnNesteSteg(props.avtaleSteg, props.aktivtSteg);
+    return (
+        <div className="nesteforrige">
+            <div className="nesteforrige__rowspacebetween">
+                {forrigeSteg && (
+                    <Route
```

Comment:
Her tror jeg det skal gå an å bruke React Router sin `Link` komponent :)
Blir noe lignende `<Link to={pathTilStegIAvtale(props.avtale.id, forrigeSteg.id)} />`.
Kan også sende med `className` som prop til link slik du allerede har gjort for å få NAV sin styling.
Mer dokumentasjon her https://reacttraining.com/react-router/web/api/Link